sklearn/manifold/t_sne.py

```diff
@@ -387,10 +387,11 @@ def trustworthiness(X, X_embedded, n_neighbors=5, precomputed=False):
         T(k) = 1 - \frac{2}{nk (2n - 3k - 1)} \sum^n_{i=1}
             \sum_{j \in U^{(k)}_i} (r(i, j) - k)
 
-    where :math:`r(i, j)` is the rank of the embedded datapoint j
-    according to the pairwise distances between the embedded datapoints,
-    :math:`U^{(k)}_i` is the set of points that are in the k nearest
-    neighbors in the embedded space but not in the original space.
+    where for each sample i, :math:`U^{(k)}_i` are all samples j that are in
+    the k-nearest neighbor of i in the output space but are the :math:`r(i,
```

Comment:
I still find this difficult. Can we just get rid of U from above and just have `max(0, r(i, j) - k)` or `max(k, r(i, j)) - k`?